test/regress_dns.c

```diff
@@ -2073,7 +2074,7 @@ dns_client_fail_requests_test(void *arg)
 	event_base_dispatch(base);
 
 	tt_int_op(n_replies_left, ==, 0);
-	for (i = 0; i < 20; ++i)
+	for (unsigned i = 0; i < 20; ++i)
```

Comment:
I think you don't need/want this `unsigned` here now

Reply:
It's a counter that never has a negative value.  Why wouldn't it be `unsigned`?

Reply:
ah, because it shadows the previous declaration... got it.